### ISSUE TITLE
Translation system .po

### ISSUE THREAD

**Issue by tkporreta** [rathena/rathena#5428]
rAthena could take the same bilingual idea that hercules has [generate-translations.c](https://github.com/HerculesWS/Hercules/blob/stable/src/plugins/generate-translations.c)

**Comment by RagnaWay** [comment]
Wow, it would be perfect for the rA to deploy that.
https://github.com/HerculesWS/Hercules/commit/330e31cc71ece055908acb1eb967b4009ebc9c46

**Comment by Cyanide0210** [comment]
Yeah this feature is very usefull it should be added definitely, there was an attempt but some commit disrupted the devs work on this so he just droped it  [#pull1829](https://github.com/rathena/rathena/pull/1829)
